### ISSUE TITLE
Cannot delete empty containers on dir backend on xfs

### ISSUE THREAD

**Issue by tych0** [lxc/incus#493]
```
# incus config show foo
architecture: x86_64
config:
  volatile.apply_template: create
  volatile.cloud-init.instance-id: 973000df-c58f-4d41-9bfa-337b9ed59fad
  volatile.idmap.base: "0"
  volatile.idmap.next: '[{"Isuid":true,"Isgid":false,"Hostid":100000,"Nsid":0,"Maprange":1000000000},{"Isuid":false,"Isgid":true,"Hostid":100000,"Nsid":0,"Maprange":1000000000}]'
  volatile.last_state.idmap: '[]'
  volatile.uuid: 280f5eed-3b12-48c8-a1c6-6c3671c7e432
  volatile.uuid.generation: 280f5eed-3b12-48c8-a1c6-6c3671c7e432
devices: {}
ephemeral: false
profiles:
- default
stateful: false
description: ""

# incus delete foo
Error: Failed deleting instance "foo" in project "default": Error deleting storage volume: Failed to set project quota for ID "10004" on "/dev/nvme1n1"
```

**Comment by stgraber** [comment]
```
root@v1:~# incus create --empty foo
Creating foo
root@v1:~# stat -f /var/lib/incus/containers/foo/
  File: "/var/lib/incus/containers/foo/"
    ID: 81000000000 Namelen: 255     Type: xfs
Block size: 4096       Fundamental block size: 4096
Blocks: Total: 5226496    Free: 5181763    Available: 5181763
Inodes: Total: 10485760   Free: 10485747
root@v1:~# incus delete foo
root@v1:~# 
```

**Comment by stgraber** [comment]
Okay, I get the issue once prjquota is enabled on the xfs mount
